src/IBM.Watson.VisualRecognition.v4/Model/Collection.cs

```diff
@@ -59,7 +59,7 @@ public class Collection
         /// Training status information for the collection.
         /// </summary>
         [JsonProperty("training_status", NullValueHandling = NullValueHandling.Ignore)]
-        public virtual TrainingStatus TrainingStatus { get; private set; }
+        public CollectionTrainingStatus TrainingStatus { get; set; }
```

Comment:
I think training status should stay as is - it was virtual bc it was readonly

Reply:
Got it!